numalogic/backtest/_clifunc.py

```diff
@@ -76,15 +76,15 @@ def generate_scores(
     ts_col_name: str,
     model_path: Union[Path, str],
     test_ratio: float,
+    use_full_data: bool = False,
 ):
     """Generate scores for the given data."""
```

Comment:
Do we need docstrings for the variables?

Reply:
They can come in later